src/stateMachine/discoveryStateMachine.hpp

```diff
@@ -74,7 +74,7 @@ class DiscoveryStateMachine final
 	};
 	struct DiscoveredEntityInfo
 	{
-		entity::Entity entity{ {},{} };
+		entity::Entity entity{ {}, {} };
```

Comment:
I still don't understand why your original file doesn't already have the missing space character, as the current version of this file does.
Are you sure you took the latest branch tip as starting point?

Reply:
@christophe-calmejane exactly from master 6d2e4210

Reply:
![image](https://user-images.githubusercontent.com/1313542/75351619-78f90080-58b9-11ea-8baa-4d3682db2516.png)

Reply:
Oh ok I see the error :)
I never develop in _master_ branch, it's only there to get a stable build.
You have to use the _dev_ branch as specified in the CONTRIBUTING.md file
I guess that minor formatting issue was fixed inbetween then.